adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -44,21 +44,21 @@ import (
 
 // getHTTPFilters generates httpFilter configuration
 func getHTTPFilters() []*hcmv3.HttpFilter {
-	extAauth := getExtAuthzHTTPFilter()
-	router := getRouterHTTPFilter()
-	lua := getLuaFilter()
+
 	cors := &hcmv3.HttpFilter{
 		Name:       wellknown.CORS,
 		ConfigType: &hcmv3.HttpFilter_TypedConfig{},
 	}
+	extAauth := getExtAuthzHTTPFilter()
+	lua := getLuaFilter()
+	router := getRouterHTTPFilter()
 
-	httpFilters := []*hcmv3.HttpFilter{
-		cors,
-		extAauth,
-		lua,
-		router,
+	logConf := config.ReadLogConfigs()
+
+	if logConf.DebugLogs.Enable {
+		return []*hcmv3.HttpFilter{cors, extAauth, getDebugLogFirstFilter(), lua, getDebugLogSecondFilter(), router}
```

Comment:
just a query; can't we get request / response headers/body from envoy access logs or trace logs?